src/applications/mhv/secure-messaging/components/Search/SearchForm.jsx

```diff
@@ -166,11 +164,12 @@ const SearchForm = props => {
         in <strong>{folderName}</strong> for{' '}
         {keyword && (
           <>
-            "<strong>{keyword}</strong>"
+            "<strong>{query.queryData.searchTerm}</strong>"
```

Comment:
I saw similar lines that have a null safety. Does this line need one as well?

Reply:
great catch!  I'll include that as well